client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -8,10 +8,38 @@ import request from '../utils/request';
 import showErrorCheckout from '../utils/show-error-checkout';
 import { buildAjaxURL } from '../../payment-request/utils';
 
-export const handlePlatformCheckoutEmailInput = ( field, api ) => {
+// Waits for the element to exist as in the Blocks checkout, sometimes the field is not immediately available.
+const waitForElement = ( selector ) => {
+	return new Promise( ( resolve ) => {
+		if ( document.querySelector( selector ) ) {
+			return resolve( document.querySelector( selector ) );
+		}
+
+		const checkoutBlock = document.querySelector(
+			'[data-block-name="woocommerce/checkout"]'
+		);
+		const observer = new MutationObserver( () => {
+			if ( document.querySelector( selector ) ) {
+				resolve( document.querySelector( selector ) );
+				observer.disconnect();
+			}
+		} );
```

Comment:
Yup JavaScript was doing something mysterious there. :laughing:  
Decided to use the callback params to avoid confusion [b71a186](https://github.com/Automattic/woocommerce-payments/pull/4638/commits/b71a1865aa4d1d2db4a3dece958cf7097c6f192d)